Automattic/wp-calypso
# Site Transfers: remove extra padding on transfer screen

Closes https://github.com/Automattic/dotcom-forge/issues/2808

## Proposed Changes

This PR removes extra padding on transfer screen on 
`http://wordpress.com/settings/start-site-transfer/{yoursite}`.

**Before**:

<img width="888" alt="Screenshot 2023-06-22 at 10 34 13 AM" src="https://github.com/Automattic/wp-calypso/assets/25575134/56a9c6dd-e859-4bae-a84b-1d0c12e30dd8">

**After**:

<img width="896" alt="Screenshot 2023-06-22 at 10 34 55 AM" src="https://github.com/Automattic/wp-calypso/assets/25575134/29f0b70a-1ee5-493c-bd08-32ebacec4590">

## Testing Instructions

* Navigate to `Settings > General > Transfer your site` on an Atomic or Simple site you can use Calypso live link)
* Select an administrator you would like to transfer your site to
* Consent to the site transfer and click on `Start transfer`
* Observe the `email sent message` and make sure there is no extra padding on the screen:

<img width="896" alt="Screenshot 2023-06-22 at 10 34 55 AM" src="https://github.com/Automattic/wp-calypso/assets/25575134/107b3133-5457-4765-9500-1dcba4cfe335">

* Check other buttons in the transfer process to make sure the padding looks good there as well

## Pre-merge Checklist
